### PR TITLE
Remove PVC when creating notebook server

### PR DESCRIPTION
**Pull request by jsitu777**
**Description of your changes:**
- create notebook server without pvc 
- specify timeout = 6 mins for one of the subprocess call

**Testing:**
- [ ] Unit tests pass
- [x] e2e tests pass
- Details about new tests (If this PR adds a new feature)
- Details about any manual tests performed

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: tests/e2e/tests/test_sanity_portforward.py [hunk 1]
```diff
@@ -220,7 +220,7 @@ def test_ack_crds(
             pods=notebook_name, namespace=DEFAULT_USER_NAMESPACE, timeout=360
         )
 
-        output = subprocess.check_output(cmd, stderr=subprocess.STDOUT).decode()
+        output = subprocess.check_output(cmd, stderr=subprocess.STDOUT, timeout=360).decode()
```
Comment (surajkota) [inline tests/e2e/tests/test_sanity_portforward.py:223]: do we need this long?

File: tests/e2e/resources/custom-resource-templates/notebook-crd.yaml [hunk 2]
```diff
@@ -30,8 +30,7 @@ spec:
       serviceAccountName: default-editor
       volumes:
       - name: ${NOTEBOOK_NAME}
```
Comment (surajkota) [inline tests/e2e/resources/custom-resource-templates/notebook-crd.yaml:32]: do we need this?
```
- name: ${NOTEBOOK_NAME}
  emptyDir: {}
```

**Review by surajkota — CHANGES_REQUESTED** [review]

File: tests/e2e/tests/test_sanity_portforward.py (hunk 1 above)
Comment (jsitu777) [reply tests/e2e/tests/test_sanity_portforward.py:223]: one minute then?

File: tests/e2e/resources/custom-resource-templates/notebook-crd.yaml (hunk 2 above)
Comment (jsitu777) [reply tests/e2e/resources/custom-resource-templates/notebook-crd.yaml]: removed.

**Pull request closed without merging**